### PR TITLE
3.13 version OHOS adaptation

### PR DESCRIPTION
**Pull request by wanghui187**

### CODE REVIEW

File: cmake/Modules/CocosBuildHelpers.cmake [hunk 1]
```diff
@@ -139,6 +140,8 @@ elseif (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set(APPLE TRUE)
     set(SYSTEM_STRING "Mac OSX")
   endif()
+elseif(OHOS)
+  set(SYSTEM_STRING "OHOS")
```
Comment (minggo) [inline cmake/Modules/CocosBuildHelpers.cmake]: Should change to `HarmonyOS Next`?

File: cmake/Modules/FindChipmunk.cmake [hunk 2]
```diff
@@ -60,6 +60,24 @@ FIND_LIBRARY(CHIPMUNK_LIBRARY
 
 set(CHIPMUNK_INCLUDE_DIRS "${CHIPMUNK_INCLUDE_DIR}")
 
+if(OHOS)
+  # 主动set路径
```
Comment (minggo) [inline cmake/Modules/FindChipmunk.cmake]: Please don't use Chinese. Please fix it in all files.

File: cmake/Modules/CocosBuildHelpers.cmake (hunk 1 above)
Comment (wanghui187) [reply cmake/Modules/CocosBuildHelpers.cmake]: fix

File: cmake/Modules/FindChipmunk.cmake (hunk 2 above)
Comment (wanghui187) [reply cmake/Modules/FindChipmunk.cmake]: fix

**Pull request merged**